backend/app/assets/stylesheets/spree/backend/shared/_forms.scss

```diff
@@ -267,19 +267,11 @@ fieldset {
       }
     }
 
-    span.or {
-      background-color: $color-1;
-      border-width: 5px;
-      margin-left: 5px;
-      margin-right: 11px;
-      position: relative;
-      z-index: 1;
-
-      -webkit-box-shadow: 0 0 0 5px $color-1;
-         -moz-box-shadow: 0 0 0 5px $color-1;
-          -ms-box-shadow: 0 0 0 5px $color-1;
-           -o-box-shadow: 0 0 0 5px $color-1;
-              box-shadow: 0 0 0 5px $color-1;
+    // Always make sure the choices at the bottom of a fieldset are spaced out
+    form + form,
+    button + a,
+    form + a {
```

Comment:
Those were the only locations that a `<span class="or">` showed up.  I can't off the top of my head confirm that the elements with `.filter-actions` only ever had two two elements in it so I'm not certain that a `* + *` selector would be safe.